templates/flavors/azure-cni-v1/patches/azure-cluster.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: infrastructure.cluster.x-k8s.io/v1beta1
+kind: AzureCluster
```

Comment:
is anything getting patched in here that's different from the default template?

Reply:
That is a good point, I am not adding anything new in this file. I initially added this with setting up multiple NICs, but since with split the effort out, I did not modify it. 

Will remove it.